src/Equinor.ProCoSys.IPO.WebApi/Startup.cs

```diff
@@ -157,6 +158,14 @@ public void ConfigureServices(IServiceCollection services)
             services.AddApplicationInsightsTelemetry(Configuration["ApplicationInsights:InstrumentationKey"]);
             services.AddMediatrModules();
             services.AddApplicationModules(Configuration);
+
+            services.AddPcsServiceBusIntegration(options =>
+            {
+                options.UseBusConnection(Configuration.GetConnectionString("ServiceBus"));
+                options.WithSubscription(PcsTopic.Project, "ipo_project");
+                options.WithSubscription(PcsTopic.Commpkg, "ipo_commpkg");
+                options.WithSubscription(PcsTopic.Mcpkg, "ipo_mcpkg");
```

Comment:
When you have made such a fancy class, you should use it. 

```c#
options.UseBusConnection().WithSubscription().WithSubscription()
```

Why is the topic named `ipo_` ?
Other systems should not care about who the receiver is.

Reply:
Fir comment: Agree
Second comment: This is the subscription name for the topic (ipos subscription for the commpkg topic), topic is commpkg.